README.md

```diff
@@ -137,38 +170,7 @@ Step 3: Starting the app after installation
 5. Open your internet browser and go to `localhost:3000`
 6. To end the application, press `control + c` in your terminal and close the terminal
 
-## Functionality
 
-This section outlines the total functionality of the app. 
-1. Viewing: The user will be able to see any Shakespeare play in readable,
-aesthetically-pleasing format.
-2. Navigation: The user will be able to navigate to any place in the script
-using a navigation bar on one side of the screen.
-
-![Navigating Play](https://user-images.githubusercontent.com/21328366/33822284-dad565da-de0b-11e7-9376-765faf6d088d.gif)
-
-3. Strikethrough: The user can cut through words and lines that they
-would like to remove from the play using highlighting or clicking on a speaker's name.
-
-![Cutting functionality](https://user-images.githubusercontent.com/21372792/33815748-b6737c72-dde8-11e7-80de-bbca75f0c62d.gif)
-
-4. Saving: The current state of the play can be saved by clicking the save button in the sidebar. From then on, when the play loads, the most recently saved version will be shown.
-5. Printing: The user van print their full, edited version of the
-play, with or without edits depending on the current toggle view mode.
-6. Analytics: The site calculates and displays information about each
-play, namely the original line number of each character, how many have been deleted, and lines remaining, shown below.
-
-![Analytics Image](https://user-images.githubusercontent.com/21372792/33813859-58cd7cc4-dddb-11e7-9cf7-c84d169c344c.png)
-
-7. Synopsis: The play synopsis can be accessed via the 'Synopsis' button in the sidebar.
-8. Full and Concise Versions: The user has the option of viewing
-either the full original version of the play, having cuts distinct but
-visible, and the concise version where cuts are removed to see context
-when necessary, shown below.
-
-![Toggle Functionality Gif](https://user-images.githubusercontent.com/21372792/33815975-a8456974-ddea-11e7-83f1-4d4958cb182e.gif)
-
-9. Clear Edits: The user can clear all the edits they've made by clicking the 'Clear Edits' button in the side bar. The page will prompt them to confirm this and then return the play to it's original unedited state.
 
 ### Use Case 
```

Comment:
Use Cases is a subsection of Functionality, so when you moved functionality to the top please include the use cases as well